tests/Core/Components/Checkbox/FluentCheckboxTests.razor

```diff
@@ -0,0 +1,184 @@
+﻿@using Microsoft.FluentUI.AspNetCore.Components.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Tests.Extensions
+@using Microsoft.FluentUI.AspNetCore.Components.Utilities
+@using System.ComponentModel.DataAnnotations
+@using Xunit;
+@inherits TestContext
+
+@code
```

Comment:
Can you add some tests for these parts of code ?

![{17D0AFC5-1EAD-4958-9CBE-3B883D13F132}](https://github.com/user-attachments/assets/aa2a9a4e-197a-4422-9d85-117959688030)

![{C4710E41-B3A7-4BF0-8893-4378853B0CB5}](https://github.com/user-attachments/assets/2c01f8b1-de5b-467d-bd8c-9a1029999138)